pixi.toml

```diff
@@ -0,0 +1,79 @@
+[project]
+authors = ["IRIS-HEP"]
+channels = ["conda-forge"]
+description = "Environments for the Analysis Grand Challenge analyses"
+name = "analysis-grand-challenge"
+platforms = ["linux-64", "osx-64", "osx-arm64"]
+version = "1.4.0"
+
+[tasks]
+
+[dependencies]
```

Comment:
These are the dependencies of the the "default" pixi "feature" (basically a composable environment chunk) that all other defined features will extend.